### DISCUSSION TITLE
cannot load plugin on Mac

### DISCUSSION THREAD

**Discussion by fdmarco3** [hbeni/fgcom-mumble#133] [Q&A] [answered]
Hello, i have downloaded fgcom mumble v.0.14.1 and put fgcom-mumble-macOS.bundle  into mumble plugin folder. Mumble version is Mumble-1.4.0~2021-05-16~g789f2d7~snapshot.x64. As i start Mumble client, i get: 
Found a non-plugin in plugin's folder: "/Applications/Mumble.app/Contents/Plugins/fgcom-mumble-macOS.bundle

I have a mac with Big Sur installed. Can you suggest what i am doing wrong?
Thankyou.

**Comment by hbeni** [comment]
Basicly this is the way this should work.
What happens when you:
- download the releases plugin asset: https://github.com/hbeni/fgcom-mumble/releases/download/v.0.14.1/fgcom-mumble-0.14.1.mumble_plugin
- Start mumble
  - Go to settings, there "Plugins"
  - Use the "install plugin" button and point it to the `.mumble_plugin`you just downloaded

**Reply by hbeni** [reply]
Ah, sorry.
Just have seen it is not the latest dev snapshot from the repo, but the last released snapshot `Mumble-1.4.0~2021-05-16~g789f2d7~snapshot.x64` from the download page.

This is not compatible with the 0.14.1 release bundle.
Please try the 0.14.0 release: https://github.com/hbeni/fgcom-mumble/releases/download/v.0.14.0/fgcom-mumble-0.14.0.zip

**Reply by fdmarco3** [reply]
Hello, i have tried with the plugin you linked, but doesn't work. If i manual install, as i launch Mumble i get the same error and if i try to install using Mumble client i get:
Failed to load plugin "fgcom-mumble-macOS.bundle" - check the plugin interface!

**Reply by hbeni** [reply]
We had the same issue just a few days ago, and Fahim was confirming it working.
Maybe the thread does help: https://github.com/hbeni/fgcom-mumble/issues/126#issuecomment-865231272

**Reply by hbeni** [reply]
@fdmarco3 Did that help?

**Reply by fdmarco3** [reply]
No. I have tried  compiling mumble dev and using fgcom-mumble. I always get the same error saying to check the plugin interface...

**Reply by hbeni** [reply]
What commit/branch of mumble specifically did you compile?

**Comment by fdmarco3** [comment]
I have followed the instructions in https://github.com/mumble-voip/mumble/blob/master/docs/dev/build-instructions/README.md
Where they say to clone 
git clone https://github.com/mumble-voip/mumble.git

**Comment by hbeni** [comment, marked as answer]
Ok, when you compile latest master branch, maybe pull the latest changes in. Today there were some changes that may help

**Reply by hbeni** [reply]
I meant both, fgcom-mumble and mumble itself

**Reply by fdmarco3** [reply]
Ok... downloaded the master source code (from github) of fgcom-mumble. From the mumble-plugin directory i launced the compile command:
make CC=g++-10 CFLAGS="-I/usr/local/opt/openssl/include/ -L/usr/local/opt/openssl/lib/" plugin
and i finally have my working **fgcom-mumble.so** i could install it in mumble using the plugin interface.
I am gonna test if it actually works..

**Reply by hbeni** [reply]
Great! It should work, and for easy Testing you could hook the radioGui to it (or try flightgear).

You should rename the `.so` to `.bundle`, so you can use the builtin updater in the future.

**Reply by hbeni** [reply]
:information_source: **As a followup for this, and for the archives:**
The Makefile now has a build target `make plugin-macOS` which will setup the make command and renaming for you.
This is also what is used by the GitHub release file builds.
